python/revng/model/migrations/migrate.py

```diff
@@ -0,0 +1,41 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from revng.model import latest_version  # type: ignore
+
+from revng.model.migrations.versions.v2 import Migrate2
+from revng.model.migrations.versions.v3 import Migrate3
+from .migration import Migration
+
+migrations: dict[int, Migration] = {
+    2: Migrate2(),
+    3: Migrate3(),
+}
+
+
+def migrate(model: dict):
+    if "Version" not in model:
+        return
+
+    version = model["Version"]
+
+    if type(version) is not int:
+        raise ValueError(
+            f".Version field in the model must be of type int but is of type {type(version)}"
```

Comment:
Why the leading `.`? Are you using some RTL system? `:D`
Same below.

Reply:
Muscle memory from Helm I guess :D It is removed now.